group-coordinator/src/main/resources/common/message/ConsumerGroupCurrentMemberAssignmentKey.json

```diff
@@ -0,0 +1,26 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// KIP-848 is in development. This schema is subject to non-backwards-compatible changes.
+{
+  "type": "data",
+  "name": "ConsumerGroupCurrentMemberAssignmentKey",
+  "validVersions": "8",
```

Comment:
Since these are new records, why are we not starting at version 0? This applies to other records too.

Reply:
That's a very good question. For the `__consumer_offsets` records, the `version` is used as an unique identifier as well. I know... This is weird. My hope is to change this in the future, possibly before KIP-848 is released, but I will stick to this pattern for now so I can focus on the core implementation first.